exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -41,6 +42,10 @@ const (
 	totalStr    = "total"
 	delimeter   = "_"
 	keyStr      = "key"
+
+	metricIntervalToCheckByteSize = 5000
```

Comment:
Can you document what metricIntervalToCheckByteSize is. From the implementation, it looks like it's not an interval but a window. Can we call it batchCheckWindowSize instead?